average.js

```diff
@@ -1,6 +1,16 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  // Filtering array to remove NaN
+  const NanRemoved = numbers.filter((num) => !Number.isNaN(num));
```

Comment:
The variables you have added (NanRemoved, etc) are in PascalCase, while the existing code is camelCase.
It's still going to work, but always good to maintain consistency.

Reply:
I was unaware about this convention, I will update the variables. Thank you sir!